web/public/js/controllers/apply-for-event-controller.js

```diff
@@ -11,6 +11,27 @@
       } else {
         $scope.showEventInfo(eventIndex, $scope.event.id);
       }
+    };
+
+    // Functions to map Service ticket types to front-end ticket types.
+    // Front end types are: 1.ninja, 2.guardian-parent 3.mentor, 4.other
+    function isNinja(user) {
```

Comment:
These functions seem like the kind of thing that might be used in a lot of places - [would this file](https://github.com/CoderDojo/cp-zen-platform/blob/master/web/public/js/controllers/header-controller.js) be a good place for something like that?